.github/workflows/neon_extra_builds.yml

```diff
@@ -32,7 +32,7 @@ jobs:
 
     steps:
       - name: Checkout
-        uses: actions/checkout@v3
+        uses: actions/checkout@v4
```

Comment:
Why bump to checkout v4 in the same commit?

Reply:
low risk and plain chore to do in separate PR